tests/run-make/no-builtins-lto/rmake.rs

```diff
@@ -0,0 +1,20 @@
+// The rlib produced by a no_builtins crate should be explicitely linked
+// during compilation, and as a result be present in the linker arguments.
+// See the comments inside the test for more details.
```

Comment:
Nit: we should clarify which file, because *this file* is the primary test file?